documentation/docs/guide/SiteEditor/Patterns/HowToUsePatterns.md

```diff
@@ -0,0 +1,25 @@
+# How to Use Patterns
```

Comment:
all though this page is ok, were you expecting formatting on this page?
![image](https://github.com/user-attachments/assets/2e339d17-f3e2-4742-b73d-d9e2286d8e55)